test/cpp/api/functional.cpp

```diff
@@ -1267,3 +1267,15 @@ TEST_F(FunctionalTest, Pad) {
     auto expected = torch::tensor({{{{0., 1., 0.}}}}, torch::kDouble);
   }
 }
+
+TEST_F(FunctionalTest, AlphaDropout) {
+  auto input = torch::randn(5000);
+  auto input_mean = input.mean();
+  auto input_std = input.std();
+
+  for (const auto rate : {0.2, 0.5, 0.8}) {
+    auto output = F::alpha_dropout(input, rate, false);
+    ASSERT_TRUE(torch::allclose(input_mean, output.mean(), 0.1));
+    ASSERT_TRUE(torch::allclose(input_std, output.std(), 0.1));
+  }
```

Comment:
Thanks so much for the awesome tests! It would be great if we could also add a test for the `F::alpha_dropout(input)` (i.e. no value passed for either `options` or `training`) use case :D

Reply:
Ah right, I missed that. Will add a test case for it

Reply:
Thanks a lot for your help!